CHANGELOG.md

```diff
@@ -7,8 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Add
+- Add `std-plonk` feature to crate [#42](https://github.com/dusk-network/plonk_gadgets/issues/42)
```

Comment:
Is there a reason why you called the feature `std-plonk`…? It should just be `std`.
It's not like you called `std-rayon`the feature in `plonk` just because the `rayon` dependency will be used in `std` no?

Reply:
Because plonk per-se doesn't require std. This just enables the `std` feature of plonk. 

`std` alone would seem that turns on `std` for some internals of this repo itself, not its deps.